contracts/cosmwasm-vm/cw-common/src/xcall_connection_msg.rs

```diff
@@ -78,4 +79,6 @@ pub enum QueryMsg {
     GetFee { nid: NetId, response: bool },
     #[returns(u64)]
     GetUnclaimedFee { nid: NetId, relayer: String },
+    #[returns(HashMap<String, String>)]
+    GetIBCConfig { nid: NetId },
 }
```

Comment:
Its better to use "GetIbcConfig" here because else we would need to query like {"get_i_b_c_config"} when using cli .